src/client/api/SConscript

```diff
@@ -8,6 +8,10 @@ def scons():
     env.AppendUnique(LIBPATH=[Dir('.')])
     denv = env.Clone()
 
+    env.Substfile('libdaos.pc.in', SUBST_DICT={'\@VERSION\@': DAOS_VERSION, '\@PREFIX\@': env.get('PREFIX')})
```

Comment:
(pylint-line-too-long) Line too long (109/80)
(lint) '\@'. String constant might be missing an r prefix.
(lint) '\@'. String constant might be missing an r prefix.
(lint) '\@'. String constant might be missing an r prefix.
(lint) '\@'. String constant might be missing an r prefix.
(pylint-no-member) Instance of 'DefaultEnvironment' has no 'Substfile' member